packages/db/test/fixtures/valid-config-files/platformatic.db.yaml

```diff
@@ -1,5 +1,5 @@
 db:
-  connectionString: {DATABASE_URL}
+  connectionString: '{DATABASE_URL}'
```

Comment:
Unfortunately, the requirement to use strings marks this change as a breaking change. I think this was a mistake on my part.

Maybe you could try supporting this by catching the parsing error, and only in that case run pupa before the parsing.